client/dashboard/emails/components/unused-mailbox-notice.tsx

```diff
@@ -0,0 +1,55 @@
+import { Link } from '@tanstack/react-router';
+import {
+	__experimentalVStack as VStack,
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { addTitanmailMailboxRoute } from '../../app/router/emails';
+import { Notice } from '../../components/notice';
+
+interface UnusedMailboxNoticeProps {
+	domains: string[];
+}
+
+const UnusedMailboxNotice = ( { domains }: UnusedMailboxNoticeProps ) => {
+	if ( ! domains?.length ) {
+		return null;
+	}
```

Comment:
Nit: With this in place I'd remove `domainsWithUnusedMailbox.length > 0` on the other file.

Reply:
Sure, removed.